### DISCUSSION TITLE
saving to new video file

### DISCUSSION THREAD

**Discussion by gbo999** [roboflow/supervision#40] [Q&A] [answered]
i am trying to apply box annotation to write to a new video file. but it return the same video the resource. i use some of the script from newest video on youtube.

`import cv2

from ultralytics import YOLO
import supervision as sv
import numpy as np
from supervision import VideoSink,VideoInfo

def main():
    
    box_annotator = sv.BoxAnnotator(
        thickness=2,
        text_thickness=1,
        text_scale=0.5
    )

    frames=[]
    model = YOLO("runs\detect\\train15\weights\\best.pt")
    for result in model.track(source='Sequence 01.mp4', show=True, stream=True, agnostic_nms=True):
        
        frame = result.orig_img
        detections = sv.Detections.from_yolov8(result)

        if result.boxes.id is not None:
            detections.tracker_id = result.boxes.id.cpu().numpy().astype(int)
        
        detections = detections[(detections.class_id != 60) & (detections.class_id != 0)]

        labels = [
            f"{tracker_id} {model.model.names[class_id]} {confidence:0.2f}"
            for _, confidence, class_id, tracker_id
            in detections
        ]

        frame1 = box_annotator.annotate(
            scene=frame, 
            detections=detections,
            labels=labels
        )
        
        frames.append(frame1)
        
        
        
    video_info = VideoInfo.from_video_path(video_path='Sequence 01.mp4')
    with VideoSink(target_path='target_video.mp4', video_info=video_info) as s:
        for frame in frames:
            s.write_frame(frame=frame)
        

if __name__ == "__main__":
    main()`

**Comment by gbo999** [comment, marked as answer]
o.k i suceeded by removing this line:    detections = detections[(detections.class_id != 60) & (detections.class_id != 0)]

**Comment by SkalskiP** [comment]
This is great! 💜
